src/utils/dif_upload.rs

```diff
@@ -1201,6 +1254,57 @@ fn create_il2cpp_mappings<'a>(difs: &[DifMatch<'a>]) -> Result<Vec<DifMatch<'a>>
     Ok(line_mappings)
 }
 
+fn resolve_ppdb(mut difs: Vec<DifMatch>) -> Vec<DifMatch> {
```

Comment:
I think this function might as well directly mutate its argument.
```suggestion
fn resolve_ppdb(difs: &mut Vec<DifMatch>) {
```